src/components/MetricsOptions/IstioMetricsOptionsBar.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { FormGroup, Toolbar } from 'patternfly-react';
+
+import { ToolbarDropdown } from '../ToolbarDropdown/ToolbarDropdown';
+import { URLParams, HistoryManager } from '../../app/History';
+import { Reporter, Direction, MetricsOptions } from '../../types/MetricsOptions';
+
+import MetricsOptionsBar from './MetricsOptionsBar';
+
+export interface IstioMetricsOptionsBarProps {
+  direction: Direction;
+}
+
+export class IstioMetricsOptionsBar extends MetricsOptionsBar<IstioMetricsOptionsBarProps> {
```

Comment:
After some time in the React world, I am start seeing that inheritance aren't one of the most preferred tools. Then, maybe a little late or maybe it could be set as ad-hoc dev-task, do you think this solution could be approached in more react-way? Like using composition of components?
Here a piece of documentation regarding this inheritance thing: https://reactjs.org/docs/composition-vs-inheritance.html#so-what-about-inheritance

Not blocking, but I guess that we all should be aware of this stuff.

Reply:
Good point, I tried to avoid inheritance but sometimes I wonder if splitting into a lot of smaller components (to allow composition) really worth a shot. In a first revision I had some inheritance between the Dashboard component and CustomMetrics and IstioMetrics components, but in this case it was quite obvious to change inheritance for composition.
Here with OptionsBar the trade is different and I suppose it would mean a split in many smaller parts to assemble.
But I'll give it another try this week :)